tests/integration/components/taxonomy-manager-test.js

```diff
@@ -225,15 +221,12 @@ module('Integration | Component | taxonomy manager', function (hooks) {
     assert.expect(5);
     this.vocabModel2.set('active', true);
 
-    this.set(
-      'assignableVocabularies',
-      resolve([this.vocabModel1, this.vocabModel2, this.vocabModel3])
-    );
+    this.set('assignableVocabularies', [this.vocabModel1, this.vocabModel2, this.vocabModel3]);
     this.set('selectedTerms', [this.termModel1, this.termModel2, this.termModel3]);
     this.set('vocabulary', this.vocabModel2);
 
     await render(hbs`<TaxonomyManager
-      @vocabularies={{await this.assignableVocabularies}}
```

Comment:
I have litterally no idea why we would have done this this way, ah... promises... 😁